### ISSUE TITLE
Account resource - document signers array

### ISSUE THREAD

**Issue by tomerweller** [stellar/go#1544]
The signers array contains all signers with a positive weight + the master signer (even if's weight is 0)

**Comment by ire-and-curses** [comment]
What array are you referring to? Response from Horizon?

**Comment by tomerweller** [comment]
Yeah, specifically this: https://www.stellar.org/developers/horizon/reference/resources/account.html
I imagine the SDK uses the same Account struct, so I'd sweep anywhere in the code that touches the it to clarify.

**Comment by ire-and-curses** [comment]
Gotcha. Yeah I guess this is another consequence of the abstraction of pretending the master signer is a signer like the others (ordinary signers are removed if set to weight 0).
